.github/workflows/build.yml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   build:
     name: "Build, Test and Publish"
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
We'll need to change the workflow definition to run this job on Linux, macOS and Windows - otherwise we'll only run the macOS, iOS and iPadOS native tests.

We'll also have to figure out how to build and publish the binaries across three OSes as well.

Reply:
As far as I know MacOS is capable of building and running tests for all targets (for linux and windows as well). But I will recheck this just in case.
In my own project I use the same approach for publishing artifacts for all platforms and it works well. I think it should be fine here as well

Reply:
You were right, tests for Linux and Windows targets are skipped (as well as tests for macOSX64). I will update the workflow to execute tests for each target on the corresponding runner. But the assembling and publication can be done on macOS for all targets

Reply:
You can find run results [here](https://github.com/OptimumCode/kaml/actions/runs/9632865172). Please, let me know if you have any suggestions on how to improve the workflow